### PR TITLE
Lphili/testimonial section

### PR DESCRIPTION
**Pull request by leeroyp**
Created a reusable expanding textbox component , button component. 
Added testimonial section to the landing page 
All components are responsive.

### CODE REVIEW

File: src/components/LandingPage/Testmonials.js [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+          name={"Enzo"}
+        />
+        <TestimonialButton
+          buttonText={"Volunteer with us"}
+          className={classes.button}
+        />
+      </div>
+      <div className={classes.div}>
+        <h2 className={classes.header}>
+        Want to help us continue to provide this program, and expand our project?
```
Comment (rishFilet) [inline src/components/LandingPage/Testmonials.js]: Text here needs to be changed:

"Are you part of an organization that works to help people who face adversity?"

**Review by rishFilet — CHANGES_REQUESTED** [review]
Nice work Leeroy, functionality is great and looks good. Just the one change of text.

I will put responsiveness in another ticket so we can close this one off

File: src/components/LandingPage/Testmonials.js (hunk 1 above)
Comment (dghazi12) [reply src/components/LandingPage/Testmonials.js]: Also, there was a typo in figma, could you remove the second "facing" in line 50: 

Screen Shot 2021-03-15 at 11 18 09 PM

**Review by dghazi12 — APPROVED** [review]
Looks good! Great job with the responsiveness.

**Review by rishFilet — APPROVED** [review]

**Pull request merged**